ISSUE_TEMPLATE.md

```diff
@@ -1,8 +1,12 @@
-Please make sure that the boxes below are checked before you submit your issue. Thank you!
+Please make sure that the boxes below are checked before you submit your issue. If your issue is an implementation question rather than a bug in Keras, please consider asking your question on the [Google Group](https://groups.google.com/forum/#!forum/keras-users) or [join the Keras Slack channel](https://keras-slack-autojoin.herokuapp.com/) rather than filing a GitHub issue.
```

Comment:
Actually, it's probably better to direct people to StackOverflow. Few people answer questions in the Google group. On SO and Slack more people are active and answering questions.